### ISSUE TITLE
PaintWidget bug on mobile phone

### ISSUE THREAD

**Issue by oeway** [Kitware/vtk-js#1314]
Hi, It's great that we can now do annotation in vtk.js. 
I am playing with the[PaintWidget example](https://kitware.github.io/vtk-js/examples/PaintWidget.html) on my mobile phone (iOS 13.3, Chrome). However, it seems the strokes doesn't end correctly, when I start a new stroke, it always connect the last point of my last stroke. Sometimes, it also connect to the lower-left corner when we start to annotate.

Besides that, in a desktop browser, if we are switched on, for example, polygonWidget, and press ESC key, it will quit the annotation mode. When one pressed ESC during polygon annotation, I think a more natural behavior is closing the current polygon annotation (so we can start a new one).

**Comment by floryst** [comment]
PaintWidget wasn't tested with mobile devices, but I can look into it. It should just involve setting the correct event handlers, as mobile uses touch events vs mouse events.

For the polygon widget, we can have that be some configurable behavior. @finetjul

**Comment by finetjul** [comment]
I agree, stopping the current polygon annotation is one thing, stopping the painting altogether is another one. The use case you describe makes sense indeed.
